pegasus/sites.v3/hourofcode.com/views/how-to/how_to_banner.haml

```diff
@@ -0,0 +1,14 @@
+#fullwidth
+  = view :header
+
+  %section.banner
+    .wrapper
+      %h1
+        =hoc_s(:hoc_how_to_top_heading)
+      %p.body-one
+        =hoc_s(:hoc_how_to_top_desc)
+      %div.show-overflow
+        .flex-container
+          %p.heading-sm.no-margin-bottom
+            =hoc_s(:hoc_how_to_top_view_guide)
+          = view :"how-to/main-nav/#{nav_audience}"
```

Comment:
This will pull in each guide's navigation order — the variable is defined here: https://github.com/code-dot-org/code-dot-org/blob/40c3e05b8da8e19c5ac29672c06a384f24cd16bc/pegasus/sites.v3/hourofcode.com/public/how-to/index-draft.haml#L8